serverless/jobs/troubleshooting/missing-metrics.mdx

```diff
@@ -0,0 +1,31 @@
+---
+meta:
+  title: Missing metrics for my Job Run
+  description: Troubleshoot missing metrics for Scaleway Serverless Jobs.
+content:
+  h1: Missing metrics for my Job Run
+  paragraph: Troubleshoot missing metrics for Scaleway Serverless Jobs.
+tags: serverless jobs troubleshooting issue missing metrics
+dates:
+  validation: 2024-12-19
+  posted: 2024-12-19
+categories:
+  - serverless
+---
+
+## Problem
+
+I can't see metrics partially or totally for my Serverless Job.
+
+## Cause
+
+Metrics for Serverless Jobs technically limited for very short Jobs under `15` seconds.
+
+## Possible solutions
+
+- If your Job takes more than around `15` seconds and metrics are still missing, it can be due to a [scheduled maintenance](https://status.scaleway.com/). If that's not the case please open a support ticket.
```

Comment:
```suggestion
- If your Job takes more than `15` seconds and metrics are still missing, it can be due to a [scheduled maintenance](https://status.scaleway.com/). If this is not the case, you can open a support ticket to report the issue. 
```